app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -206,12 +207,28 @@ class DefaultBrowserToolbarController(
                         .show()
                 }
             }
-            ToolbarMenu.Item.AddToHomeScreen, ToolbarMenu.Item.InstallToHomeScreen -> {
+            ToolbarMenu.Item.AddToHomeScreen -> {
                 activity.settings().installPwaOpened = true
```

Comment:
Also while you're here, do you mind moving this line inside the `isInstallable()` if statement?